### PR TITLE
New endpoint to get the relayer version

### LINKED ISSUE #120: [SupportV2] Add commit hash and version to /info service
The commit hash and ref name ( branch and tag)  must be retrieved from github action context and  saved as env variable during image build : https://github.com/orgs/community/discussions/26686#discussioncomment-3396593
```
- name: Create docker image
  run: ./docker-build.sh ${{ github.head_ref }}.${{ github.sha }}
```

### PR DESCRIPTION
**Pull request by LyzeOfKiel**
Changes:
* Add `/version` endpoint. Version is set via `RELAYER_VERSION` env. This variable is automatically set during CI build stage in the following format: `(branch_name|tag_name)@commit_hash`. Example response:
```json
{
    "version": "feature/relayer-build-version@6e50bafad48bfad3d20a605b7b9f370309336072"
}
```
Closes #120

### CODE REVIEW

**Review by akolotov — COMMENTED** [review]
Is it possible to add more sophisticated logic:
- use the release tag if it is associated with the corresponding commit
- use the commit hash if there is no associated tag

?

**Comment by LyzeOfKiel** [comment]
> Is it possible to add more sophisticated logic:
> 
> * use the release tag if it is associated with the corresponding commit
> * use the commit hash if there is no associated tag
> 
> ?

`github.ref_name` will be equal to [tag](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context) instead of branch name if it tag push triggers the workflow. Do you suggest to remove commit hash in this case?

**Comment by akolotov** [comment]
> Do you suggest to remove commit hash in this case?

No, that's fine.

File: zp-relayer/endpoints.ts [hunk 1]
```diff
@@ -278,6 +278,12 @@ function getParamsHash(type: 'tree' | 'transfer') {
   }
 }
 
+function relayerVersion(req: Request, res: Response) {
+  res.json({
+    version: config.version,
```
Comment (k1rill-fedoseev) [inline zp-relayer/endpoints.ts:283]: I think it's better to put tag/branch and commit hash as separate values and let the UI decide how to render them

**Comment by LyzeOfKiel** [comment]
Now, response looks like this:
```json
{
    "ref": "feature/relayer-build-version",
    "commitHash": "6e50bafad48bfad3d20a605b7b9f370309336072"
}
```

**Review by k1rill-fedoseev — APPROVED** [review]

**Review by akolotov — APPROVED** [review]

**Comment by akolotov** [comment]
Will be merged as soon as https://github.com/zkBob/zeropool-relayer/pull/116 is closed.

**Comment by LyzeOfKiel** [comment]
If the current response structure is fine I will update README later

**Pull request merged**